terraform-vpc/securitygroup.tf

```diff
@@ -0,0 +1,97 @@
+resource "aws_security_group" "website" {
+    name = "website"
+    description = "Allow inbound traffic"
+    vpc_id = "${aws_vpc.da-wordpress-vpc.id}"
+
+    tags {
+        Name = "website"
+    }
+
+    ingress {
+        from_port = 22
+        to_port = 22
+        protocol = "tcp"
+        cidr_blocks = ["0.0.0.0/0"]
+    }
+
+    ingress {
+        from_port = 80
+        to_port = 80
+        protocol = "tcp"
+        cidr_blocks = ["0.0.0.0/0"]
+    }
+
+    ingress {
+        from_port = 443
+        to_port = 443
+        protocol = "tcp"
+        cidr_blocks = ["0.0.0.0/0"]
+    }
+
+    egress {
+        from_port = 0
+        to_port = 0
+        protocol = "-1"
+        cidr_blocks = ["0.0.0.0/0"]
+    }
+
+}
+
+resource "aws_security_group" "database" {
+    name = "database"
+    description = "Allow inbound traffic"
+    vpc_id = "${aws_vpc.da-wordpress-vpc.id}"
+
+    tags {
+        Name = "database"
+    }
+
+    ingress {
+        from_port = 22
+        to_port = 22
+        protocol = "tcp"
+        security_groups  = ["${aws_security_group.website.id}"]
+    }
+
+    ingress {
+        from_port = 3306
+        to_port = 3306
+        protocol = "tcp"
+        security_groups  = ["${aws_security_group.website.id}"]
+    }
+
+   
+    egress {
+        from_port = 0
+        to_port = 0
+        protocol = "-1"
+        cidr_blocks = ["0.0.0.0/0"]
+    }
+
+}
+
+resource "aws_security_group" "efs" {
+    name = "efs-traffic"
+    description = "Allow inbound traffic"
+    vpc_id = "${aws_vpc.da-wordpress-vpc.id}"
+
+    tags {
+        Name = "db-traffic"
+        Project = "wordpress"
+    }
+
+    ingress {
+        from_port = 2049
+        to_port = 2049
+        protocol = "tcp"
+        cidr_blocks = ["0.0.0.0/0"]
+    }
+   
+    egress {
+        from_port = 0
+        to_port = 0
+        protocol = "-1"
+        cidr_blocks = ["0.0.0.0/0"]
+    }
+
+}
```

Comment:
Not sure if this should be created at this moment, specially with those rules.

Reply:
Agreed,
What would be the best way to connect the EFS to the containers for the Static-WP Content ?

Reply:
Handle this problem in the ECS PR. This is the VPC, so you only need to get the VPC infrastructure in place.